pkg/run/run.go

```diff
@@ -61,10 +56,10 @@ func (b *DockerRunner) Run(config *api.Config) error {
 
 	go docker.StreamContainerIO(errReader, nil, glog.Error)
 	go docker.StreamContainerIO(outReader, nil, glog.Info)
-	rerr := b.ContainerClient.RunContainer(opts)
-	if e, ok := rerr.(errors.ContainerError); ok {
-		return errors.NewContainerError(config.Tag, e.ErrorCode, errOutput)
-	}
 
+	err := b.ContainerClient.RunContainer(opts)
+	if e, ok := err.(errors.ContainerError); ok {
+		return errors.NewContainerError(config.Tag, e.ErrorCode, "")
```

Comment:
@gabemontero this is a question to you... why do we replace a `ContainerError` with another `ContainerError`?